ui/component/publishForm/view.jsx

```diff
@@ -293,27 +294,61 @@ function PublishForm(props: Props) {
   }, [modal]);
 
   // move this to lbryinc OR to a file under ui, and/or provide a standardized livestreaming config.
-  function fetchLivestreams(channelId, signature, timestamp) {
-    setCheckingLivestreams(true);
-    fetch(`${LIVESTREAM_REPLAY_API}/${channelId}?signature=${signature || ''}&signing_ts=${timestamp || ''}`) // claimChannelId
-      .then((res) => res.json())
-      .then((res) => {
-        if (!res || !res.data) {
-          setLivestreamData([]);
+  function fetchLivestreams(channelId, signature, timestamp, channelName) {
+    (async function() {
+      setCheckingLivestreams(true);
+
+      const newEndpointUrl = `${NEW_LIVESTREAM_REPLAY_API}?channel_claim_id=${channelId}` +
+        `&signature=${signature}&signature_ts=${timestamp}&channel_name=${channelName}`;
+
+      const responseFromNewApi = await fetch(newEndpointUrl);
+
+      const data = (await responseFromNewApi.json()).data;
+
+      const newData = data.map(function(dataItem) {
+        if (dataItem.Status === 'ready') {
+          return {
+            data: {
+              fileLocation: dataItem.URL,
+              fileDuration: (dataItem.Duration / 1000000000).toString(), // convert nanoseconds to second
+              thumbnails: [dataItem.ThumbnailURL],
+              uploadedAt: dataItem.Created,
+            },
+          };
```

Comment:
### Replay code could be sending undefined data
`map` doesn't feel like the right choice -- it is typically used for 1:1 transformation, so this code would leave blank entries if status is not `ready`:

<img width="254" alt="image" src="https://user-images.githubusercontent.com/64950861/155063980-e5e11ce6-a9c3-4ca2-a871-51dab2bf7afd.png">

A simple for-loop + `push` feels like the right tool here.

Later on, we see this:
```
// TODO: there is no guarantee this code doesn't blow up
const amountOfUploadsToRemove = newData.length;
// TODO: use a pure functional method instead
newOldApiData.splice(0, amountOfUploadsToRemove);
```

<img width="553" alt="image" src="https://user-images.githubusercontent.com/64950861/155064864-677cc1a6-e85d-4cb9-8783-aa4f9817fc55.png">

(1) Undefined is sent.
(2) Is it ok to splice it that way rather than de-dup?